docs/feature-flagging/experiment-assignment.md

```diff
@@ -23,3 +23,51 @@ Click "Create Experiment" on the experiment allocation to create an experiment.
 Once assignments have begun, they will need to be logged to your data warehouse to be analyzed by Eppo. All SDKs take in a callback function that is called with the assignment parameters (`subject_id`, `experiment_id`, `timestamp`, etc.) passed in when assignments are made. That function is completely managed by you - the only requirement is that the assignment data eventually makes it to your warehouse. 
 
 Congratulations on setting up your first Eppo randomized experiment! Refer to our [analysis setup guide](/experiment-quickstart) to connect Eppo to your data warehouse and easily analyze the results.
+
+## Running no/low code experiments in Eppo
+
+Often times there is a need for teams to be able to release changes quickly without a code release, or there are frequently tested parts of a site or app where teams would like to execute changes in a self-service manner. Running experiments without having to release additional code is easy to do in Eppo by using a JSON flag type to parameterize parts of the site or app where frequent changes are made and creating additional experiment allocations on the same flag.
+
+### Creating multiple experiments on the same flag
+
+For this example let’s use the following feature flag in Eppo. Right now there are 2 allocations, one is a feature gate that prioritizes sending any internal users to the treatment variation, and the other is an experiment allocation that evaluates whether the other users meet the audience criteria, and if they do split them evenly between the control and treatment variations on 50% of the total traffic.
```

Comment:
"the other users" confused me on first reading. I would say "the remaining users". Splitting this into multiple sentences, or a bulleted list, might also help with readability.

Reply:
updated for readability